src/aux-server/aux-web/aux-player/PlayerApp/PlayerApp.ts

```diff
@@ -1401,6 +1426,17 @@ export default class PlayerApp extends Vue {
         this.setTitleToID();
     }
 
+    private async _getScriptIssues(
+        e: GetScriptIssuesAction,
+        simulation: BrowserSimulation
+    ) {
+        const helpers = await import('../../shared/MonacoHelpers');
+        const bot = simulation.helper.botsState[e.botId];
+        const issues = await helpers.getScriptIssues(simulation, bot, e.tag);
+
+        simulation.helper.transaction(asyncResult(e.taskId, issues));
+    }
```

Comment:
@TroyceGowdy This function doesn't contain any error handling for the event, can you use the try/catch block from above?